src/pages/home/components/Sidebar/Sidebar.tsx

```diff
@@ -34,41 +38,68 @@ function Sidebar(): ReactNode {
   });
 
   return (
-    <AccordionTab hideCloseButton title="Select Social Media">
-      <div className={scss.content}>
-        <InputSearch
-          error={false}
-          onChange={(currentValue) => setValue(currentValue as string)}
-          placeholder="Search for social media"
-          ref={inputSearchRef}
-        />
+    <div className={scss.container}>
+      <Button onClick={() => setMobileIsOpen(!mobileIsOpen)} variant="outlined">
```

Comment:
Quando é pra fazer a negacao de algo assim, a melhor coisa é negar o estado previo dele
https://medium.com/geekculture/using-prevstate-with-react-basic-and-complex-f7f8ab4ce1b6
```suggestion
      <Button onClick={() => setMobileIsOpen((prev) => !prev} variant="outlined">
```

Reply:
Sim sim, é verdade, alterei.